src/plugins/console/public/application/containers/editor/monaco_editor_actions_provider.ts

```diff
@@ -579,6 +582,144 @@ export class MonacoEditorActionsProvider {
     return model.getValue();
   }
 
+  public async readStreamToString(reader) {
```

Comment:
Probably we can do it better, for the demo I just brute-forced decoding the stream in js